CONTRIBUTING.md

```diff
@@ -230,12 +231,14 @@ Each source file needs to have this header:
     // See the License for the specific language governing permissions and
     // limitations under the License.
 ```
-Note: `DATE` is either a year or a range of years with the first and last years of the range separated by a comma. So for example: "2004" or "2000, 2004". The first year is when the contents of the file were first created and the last year is when the contents were last modified.
+Note: `DATE` is either a year or a range of years with the first and last years of the range separated by a dash. So for example: "2004" (initial and last contribution in the same year) or "2000 - 2004". The first year is when the contents of the file were first created and the last year is when the contents were last modified. The year of the last contribution needs to be updated the content was modified in a new year. An option is to add a year by a comma for the case that a contributor did not modify in one year.
 
 Example:
 
-```
-    // Copyright (c) 2018, 2020 by ACME Corp, Globex. All rights reserved.
+```cpp
+    // Copyright (c) 2019 - 2020, 2022 by Contributor1. All rights reserved.
+    // Copyright (c) 2020 - 2022 by Contributor2. All rights reserved.
+    // SPDX-License-Identifier: Apache License 2.0
```

Comment:
I think this should be `// SPDX-License-Identifier: Apache-2.0` according to https://spdx.org/licenses/